packages/uiweb/src/lib/components/space/SpaceWidget/LiveWidgetContent.tsx

```diff
@@ -445,4 +598,137 @@ const RequestsCount = styled.div`
   padding: 2px 4px;
   border-radius: 4px;
   font-size: 12px;
-`;
\ No newline at end of file
+`;
+
+const fadeOutAnimation = css<{
+  showEmojiContainer?: boolean;
+  showPfpEmojiContainer?: boolean;
+}>`
+  animation: ${keyframes`
+    from {
+      opacity: 1;
+      transform: translateY(0%);
+    }
+    to {
+      opacity: 0;
+      transform: translateY(50%);
+    }
+  `} 0.3s ease-in-out;
+  animation-fill-mode: forwards;
+`;
+
+const EmojiContainer = styled.div<{
+  showEmojiContainer: boolean;
+}>`
+  position: relative;
+  bottom: -20px;
+  width: 220px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 12px;
+  border-radius: 99px;
+  border: 1px solid ${(props) => props.theme.borderColor};
+  background: ${(props) => props.theme.bgColorSecondary};
+
+  animation: ${keyframes`
+    from {
+      opacity: 0;
+      transform: translateY(50%);
+    }
+    to {
+      opacity: 1;
+      transform: translateY(0%);
+    }
+  `} 0.3s ease-in-out;
+
+  ${(props) => (!props.showEmojiContainer ? fadeOutAnimation : '')}
+
+  & > img {
+    cursor: pointer;
+  }
+
+  & > img:hover {
+    transform: scale(1.2);
+  }
+
+  & > img:active {
+    transform: scale(1.1);
+  }
+`;
+
+const SpeakerEmojiContainer = styled.div<{ showEmojiContainer: boolean }>`
+  position: relative;
+  bottom: -20px;
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 12px;
+  border-radius: 99px;
+  border: 1px solid ${(props) => props.theme.borderColor};
+  background: ${(props) => props.theme.bgColorSecondary};
```

Comment:
```suggestion
  background: ${(props) => props.theme.bgColorSecondary};
  transition: 0.2s ease-in-out;
```

add a small transition as above for a smooth hover effect